utils/controleval.py

```diff
@@ -138,6 +213,11 @@ def create_parser():
         required=True)
     statsparser.add_argument(
         "-p", "--product", help="product to check has required references")
+    statsparser.add_argument(
+        "-o",
+        "--output_format",
```

Comment:
I should have suggested this earlier, sorry for not catching it. But using `-` instead of `_` is probably more consistent here.

I think my original suggestion used the underscore instead of a dash.

Reply:
I think that's a good suggestion, thanks for the throughout reviews!